packages/dcos-integration-test/extra/test_dcos_log.py

```diff
@@ -32,7 +32,7 @@ def test_log_text(cluster):
 
         # expect 10 lines
         lines = filter(lambda x: x != '', response.content.decode().split('\n'))
-        assert len(list(lines)) == 10, 'Expect 10 log entries. Got {}. All lines {}'.format(len(lines), lines)
+        assert len(list(lines)) == 10, 'Expect 10 log entries. Got {}. All lines {}'.format(len(list(lines)), lines)
```

Comment:
This isn't correct either, the second `len(list(lines))` will always evaluate to `0` since the `lines` generator has been consumed by the first `list()` in the assertion. Should amend the above line to be `lines = list(filter(...))` and then the two `len()`s in this line will work.